contracts/Staking/Vesting.sol

```diff
@@ -53,30 +48,36 @@ contract Vesting is OwnableUpgradeable {
         // to prevent melicious users form cloging user's schedules
         require(msg.sender == vestingScheduler, "Only vesting scheduler can create vesting schedules");
 
-        vestingSchedules[_receiver].push(VestingSchedule(block.timestamp, block.timestamp.add(vestingTimeInSeconds), _amount_d18, 0));
+        vestingSchedules[_receiver].push(VestingSchedule(block.timestamp, block.timestamp + vestingTimeInSeconds, _amount_d18, 0));
 
         vestedToken.safeTransferFrom(fundsProvider, address(this), _amount_d18);
 
         emit ScheduleCreated(_receiver, _amount_d18);
     }
 
     function claim(uint256 from, uint256 to) external {
+        require(from < to, "Vesting: 'to' must be larger than 'from'");
```

Comment:
what if `from = to`? to claim only one vesting schedule

Reply:
It's not supported even today on the contract. So I did not want to change it for now in this PR.

To be honest it's not something we'll really use as we'll still send from 0 until let's say 100, so not an urgency changing it

Reply:
agree